parser/ast.go

```diff
@@ -33,13 +33,13 @@ type Assignment struct {
 	NamePos    scanner.Position
 	Value      Expression
 	OrigValue  Expression
-	Pos        scanner.Position
+	Equals     scanner.Position
```

Comment:
Maybe EqualsPos? Equals feels very non-specific to me. Colon below is a little better, but could still have a Pos suffix